clang/test/CXX/dcl.dcl/dcl.attr/dcl.attr.grammar/p2-1z.cpp

```diff
@@ -12,5 +12,5 @@
 [[using clang:]] extern int n; // ok
 [[using blah: clang::optnone]] extern int n; // expected-error {{attribute with scope specifier cannot follow}} expected-warning {{only applies to functions}}
 
-[[using clang: unknown_attr]] extern int n; // expected-warning {{unknown attribute}}
```

Comment:
This shows the exact problem here that I have above.  We KNOW 'clang' (in fact, we OWN it).  We should diagnose this as an unknown attribute, not as an unknown namespace.